projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.html

```diff
@@ -291,15 +343,17 @@ <h2 class="row no-gutters section-header-heading">
             >
               <ng-container
                 *ngIf="nextCheckoutStep$ | async as nextCheckoutStep"
-                ><button
+              >
+                <button
                   class="primary-button btn-block"
                   type="button"
+                  [disabled]="!(isCartTransferAllowedForSeller$ | async)"
```

Comment:
Here is one more call of async pipe on the same variable. Maybe you could do the same as on the `AgentSearchListComponent` template?